pandas/core/generic.py

```diff
@@ -5716,24 +5716,75 @@ def clip_upper(self, threshold, axis=None, inplace=False):
 
     def clip_lower(self, threshold, axis=None, inplace=False):
         """
-        Return copy of the input with values below given value(s) truncated.
+        Trim the caller's values below a given `threshold`.
+
+        Elements below the `threshold` will be changed to match the
+        `threshold` value(s).
 
         Parameters
         ----------
-        threshold : float or array_like
-        axis : int or string axis name, optional
+        threshold : float or array-like
+            Lower value(s) to which the input value(s) will be trimmed.
+        axis : {0 or 'index', 1 or 'columns', None}, default None
             Align object with threshold along the given axis.
         inplace : boolean, default False
-            Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+            Whether to perform the operation in place on the data.
 
-        See Also
-        --------
-        clip
+            .. versionadded:: 0.21.0
 
         Returns
         -------
-        clipped : same type as input
+        Series or DataFrame
+            Same type as caller.
```

Comment:
For this method may be it's a bit obvious, but for consistency I'd say in the description what this method returns (e.g. Original data with values trimmed). The note about the time is all right (not sure if other methods have it as part of the type or the description).

Reply:
Would any of these work?

Series or DataFrame (same type as caller)
   Original data with values trimmed.

Series or DataFrame
   Original data (in same type as caller) with values trimmed .

Reply:
They look great. If you want to take a look at the latest merged pull requests, I think this happens in few, and may be there is a way more standard than another. If you don't find any, from my side you can choose whatever you want.

Reply:
Based on what I see in the code, this seems to be the most common format

type of caller
    Original data with values trimmed.

I was thinking that there would be some value in creating PRs for standarizing these kind of things. For example, do one that fixes the axis parameters in ALL methods to match the format agreed in the discussions in this sprint. Another one could be done to make sure there is no period after the versionadded (which a lot of them have it, probably due to the validating script complaining if it was not there). Would this be a good idea? Who should I contact if I want to take care of some of these?

Reply:
Feel free to do it. I'd probably wait until most of the PRs from the sprint are closed, to avoid conflicts. And more than contacting someone, you can create an issue (or one per change) with what you're planning to do, you can get some discussion and feedback before doing all the work.